src/react-app-env.d.ts

```diff
@@ -1 +1,43 @@
 /// <reference types="react-scripts" />
+
+interface Post {
+  id: number;
+  userId: number;
+  title: string;
+  body: string;
+  createdAt: string;
+  updatedAt: string;
+}
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  phone: string;
+  website: string;
+  createdAt: string;
+  updatedAt: string;
+  address: Adress;
```

Comment:
```suggestion
  address: Address;
```
by the way, do you really need this?

Reply:
Yes, I don't need it here, but Dima at the lecture advised to describe types at the start of development for future use.